src/OSPSuite.Core/Services/CurveNamer.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Chart;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.Mappers;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Core.Services
+{
+   public interface ICurveNamer
+   {
+      string CurveNameForColumn(ISimulation simulation, DataColumn dataColumn);
+      IReadOnlyList<ICurve> CurvesWithOriginalName(ISimulation simulation, IEnumerable<ICurveChart> curveCharts);
```

Comment:
Not sure exactly as to what that method is doing. Either rename the method or add a comment explaining what Original means in regards to the simulation